zebra-rpc/Cargo.toml

```diff
@@ -8,3 +8,9 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+
+jsonrpc-core = "18.0.0"
+jsonrpc-core-client = "18.0.0"
```

Comment:
This is needed for the `#[rpc]` macro to work.

Reply:
Ok, this is good, thanks. https://github.com/ZcashFoundation/zebra/pull/3589/commits/5d744b50ffd63d40fd3aad4ff54500d1fa27b007